developing-without-a-build/article.md

```diff
@@ -0,0 +1,257 @@
+# Developing without a build
+In the early days of web development all we needed was a simple file editor and a web server. The development experience was really simple, and easy for a newcomer to understand and get started. Web development has changed a lot since then, the complexity of the tools we use for development has grown just as much as the complexity of the things that we're building on the web.
+
+These tools can be really complex, projects can set up systems which transform different syntaxes and dialects into bundles we can run in the browser. Source maps need to be configured carefully between the different tools, so that we can still recognize and debug our code in the browser.
+
+Imagine a new person coming into web development, even experienced developers, they first need to learn about a lot of different tools before they can get to actual development. Their IDE and linter don't understand the syntax of this framework that was recommended to them by a friend. Installing a few extra plugins solved that. After that they ran into some more errors... Turns out they wanted to keep it simple and not use typescript, but they didn't configure the babel plugins for decorators correctly in the right order so the project didn't compile. Oops!
+
+It sounds exaggerated, but these things are really happening to people. I feel that's really a shame. One of the key selling points of the web is that's an easy and open format. It should also be easy to just get started right away without a lot of ceremony.
```

Comment:
```suggestion
It may sound exaggerated, but this experience is common to a lot of developers. You may have jumped through similar hoops yourself only to think "this is just the way modern web development is".

I feel that's really a shame. One of the key selling points of the web is and always was that's an easy and open format. It should be easy to just get started right away without a lot of configuration, tooling, pomp, and ceremony.
```